README.md

```diff
@@ -73,6 +73,14 @@ Line Mode command; the selection is moved linewise.)
 - `.moveAndRenameFile` or `:Move`: Move and Rename the current file. Keeps the
 old name if the new path ends with `/`. Works like the UNIX `mv` command. Best
 used with [autocompletion of directories](#autocompletion-of-directories).
+- `.trashFile`: Trash the current file.
+	- Use `trashCmd` to specify an external trash command. It defaults to
+`gio trash` on *Linux*, `trash` on *Mac* and *Windows*.
+	- Otherwise specify `trashLocation` to move the file to that directory. It
+defaults to `$HOME/.Trash`. This does NOT trash the file the usual way. The
+trashed file is NOT restorable to its original path.
+	- For backwards compatibility, the default behavior on *Mac* moves files to
+`$HOME/.Trash`.
```

Comment:
I'm not familiar with *Mac*, but I read threads like [macos - How to move files to trash from command line? - Ask Different](https://apple.stackexchange.com/q/50844).

In the *iCloud* case, can you restore the file to its original location from the recycle bin, after moving them in that way? Otherwise I wouldn't want to confuse any users.

Reply:
No, you can only manually move them back in both cases. The iCloud trash is exception is only there cause under the hood, mac uses two trash locations it merges into one when the user opens the trash. 

I am on Mac, and without using an extra Cli, this is really the best solution. "on mac, it is moved to the trash" is really the simplest way to describe it for users.

Reply:
Guess this is the only thing left now.

I feel uncomfortable not giving a warning here. "moved to trash" implies this is following all the standards, moving a file to trash the same way as say in a file manager. It is not the case if it cannot be restored.

The user can just choose to not install a dependency, and move a file to some directory. But they should be warned of the consequences of doing so.

It's not just *Windows*. A trash utility is required on each and every OS.

Reply:
> But they should be warned of the consequences of doing so.

On mac, the only consequences are that the deletion operation fails, and nothing happens. So it's not that *big* of a deal. But I kinda get your point. Let's merge this, and I'll add some notes regarding mac in a bit.

Reply:
Ah whoops I haven't pushed my latest changes yet. I'll file a new PR with them.